docs/2023/DIGIT/Vehicle Tracker/updates/2023-07-28.md

```diff
@@ -4,13 +4,18 @@ author: Achintya
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- Added some changes to the Vehicle Tracking feature
+- Added some implementation for storage using Hive
```

Comment:
Add these in ticked checkboxes instead of just points showing which Milestones you achieved and which not

Reply:
ok added them accordingly